pygeoapi/linked_data.py

```diff
@@ -201,5 +215,8 @@ def geojson2geojsonld(config, data, dataset, identifier=None):
                 feature['id'] = featureId
             else:
                 feature['id'] = '{}/{}'.format(data['id'], featureId)
+    else:
+        jsonld_data["geometry"] = {"schema:encodingFormat": "application/geo+json",
+                     "schema:url": data['id'] + '?f=json'}
```

Comment:
Is this really correct? Linking to the JSON representation of itself isn't really the `geometry`, since it would be the entire object, including the properties that were already transported in the JSON-LD response. Perhaps pygeoapi needs an alternate endpoint or query paramter (i.e. `geom=only` or something...) that is *just* geometry for a collection or feature, so this will work as expected.

I think it's also worth discussing whether this should be an option alongside including the geometry in-line, and then which behaviour should be the default.

Reply:
This is just a problematic aspect of converting GeoJSON into linked data. It's certainly not valid JSON-LD to include the geometry in line. It's also not really valid to link directly to a non-semantic resource as the object of a triple. This is a compromise that I'm really not sure is worth pursuing. Just dropping the geometry in the JSON-LD representation is probably fine, IMHO.

Reply:
OK, well it should probably have some kind of link to the complete GeoJSON representation then. Perhaps later on we can add logic for computing a https://schema.org/GeoShape, or convert to WKT.

For now, would you be happy if we included the geometry in-line if it's a Point, since that is valid JSON-LD, and omitting it otherwise?

Reply:
Yes -- if point, I think it's a good idea to leave it in. For other geometry types, I think the best way to do it is to have a blank node with a link to the complete feature collection up at the root of the document. 

There's already the "rel": "alternate" that points to the geoJSON representation, so we are kind of covered. I'm not seeing a context for the links JSON keys witha little googling, maybe that exists?

Reply:
Can you please provide an example of what the non-point GeoJSON-LD should look like with the blank node? I was just planning to drop the `geometry` field; not sure if that's what you had in mind.

The `links` will be dropped in the GeoJSON-LD output as well, at least until someone proposes what these should be mapped to in some vocabulary.

Reply:
Sure thing, and I agree dropping the geometry field is a valid approach as well.  Here is what it looks like:

![image](https://user-images.githubusercontent.com/30809186/85803309-d8a2d380-b714-11ea-90b0-2b461de1f1e9.png)
 
The JSON is not a direct mapping, although the type is Feature it's mapping to geometry which is understandably not ideal.